app/experimenter/nimbus-ui/src/lib/visualization/mocks.tsx

```diff
@@ -203,3 +203,180 @@ export const mockAnalysis = (modifications = {}) =>
     },
     modifications,
   );
+
+export const mockIncompleteAnalysis = (modifications = {}) =>
```

Comment:
Since the only diff between `mockAnalysis` and `mockIncompleteAnalysis` is `retained`, you could just call `mockAnalysis` inside this function and send in `retained` as modifications? If that's too messy maybe a comment noting what makes an analysis "incomplete" will do.

Reply:
It was a bit messy using the modifications since the object is quite nested. I added a comment instead.